### ISSUE TITLE
Proposal: Simply configuration options

### ISSUE THREAD

**Issue by andrewkroh** [elastic/libbeat#264]
I noticed a few areas of the configuration files that I think could be simplified. This is a proposal to make some changes and I would like to get feedback prior to changing anything.

1. Remove boolean `enabled` and `disabled` options. Use commenting as an alternative.

2. Explicitly declare a logging output with `logger: syslog` or `logger: file` or `logger: [syslog, file]`. Currently loggers are configured as multiple boolean options (i.e. `to_syslog: true`, `to_files: true`).

3. Rename the command line option of `-test` to `-configtest`. The more clearly identifies the purpose of the flag and aligns with logstash's analogous flag.

**Comment by urso** [comment]
+1 for points 1 and 3

point 2 won't work well with yaml unless we start parsing ourselves. Would rather like to see:

    logging:
        syslog:

        file:
            ...

**Comment by andrewkroh** [comment]
I agree, it would be better to do it as @urso suggested. It is also more consistent with how the outputs are configured.

I added a forth proposed change above.

**Comment by mrkschan** [comment]
Can we have (1) asap?

**Comment by ruflin** [comment]
I'm +1 on 1-4, but for the 2 the suggestion from @urso as it will allow more logging specific configs (if needed).
